go/multiorch/recovery/actions/bootstrap_shard.go

```diff
@@ -86,7 +86,16 @@ func (a *BootstrapShardAction) Execute(ctx context.Context, shardID string, data
 		"database", database,
 		"candidate", candidate.MultiPooler.Id.Name)
 
-	// Step 3: Initialize the candidate as an empty primary with term=1
+	// Step 3: Set pooler type to PRIMARY before initializing
+	changeTypeReq := &multipoolermanagerdatapb.ChangeTypeRequest{
+		PoolerType: clustermetadatapb.PoolerType_PRIMARY,
+	}
+	_, err = a.rpcClient.ChangeType(ctx, candidate.MultiPooler, changeTypeReq)
+	if err != nil {
+		return mterrors.Wrap(err, "failed to set pooler type to PRIMARY")
+	}
+
```

Comment:
This could also be done _after_  InitializeEmptyPrimary instead of before. That seems safer to me. Thoughts?